Editorial review: Document element-scoped view transitions (#44217)
mdn/content: files/en-us/web/api/viewtransition/transitionroot/index.md

---
title: "ViewTransition: transitionRoot property"
short-title: transitionRoot
slug: Web/API/ViewTransition/transitionRoot
page-type: web-api-instance-property
browser-compat: api.ViewTransition.transitionRoot
---

{{APIRef("View Transition API")}}

The **`transitionRoot`** read-only property of the

> {{domxref("ViewTransition")}} interface is a reference to the root {{domxref("Element")}} of the [view transition](/en-US/docs/Web/API/View_Transition_API/Using) scope.

Reviewer: is the line break intentional?

Author: No. Removed ;-)

## Value

An {{domxref("Element")}}.

> - For document-scoped view transitions, `transitionRoot` will be a reference to the {{domxref("Document.documentElement")}} property, which for standard web documents is a reference to the {{htmlelement("html")}} element.

Reviewer: ```suggestion
- For document-scoped view transitions, `transitionRoot` is a reference to the {{domxref("Document.documentElement")}} property, which for standard web documents is a reference to the {{htmlelement("html")}} element.
```

Author: updated

> - For [element-scoped view transitions](/en-US/docs/Web/API/View_Transition_API/Using_element-scoped), `transitionRoot` will be a reference to the `Element` that {{domxref("Element.startViewTransition()", "startViewTransition()")}} was called on to create the view transition.

Reviewer: ```suggestion
- For [element-scoped view transitions](/en-US/docs/Web/API/View_Transition_API/Using_element-scoped), `transitionRoot` is a reference to the `Element` that {{domxref("Element.startViewTransition()", "startViewTransition()")}} was called on to create the view transition.
```

Author: updated

## Examples

### Basic usage

> 

Reviewer: same - would be nice to have a one-liner description

Author: added
